lib/octoicons_web/src/index.js

```diff
@@ -0,0 +1,36 @@
+var data = require('../../build/data.json')
+
+var icons = document.querySelectorAll("[data-icon]");
+
+icons.forEach(function(icon){
+  var name = icon.getAttribute("data-icon")
+  var width = icon.getAttribute("width")
+  var height = icon.getAttribute("height")
+  var icon_data = data[name]
+  icon_data.options = {
+    "version": "1.1",
+    "width": width || icon_data.width,
+    "height": height || icon_data.width,
+    "viewBox": "0 0 " + icon_data.width + " " + icon_data.width,
+    "class": "octicon octicon-" + name,
+    "aria-hidden": "true"
+  }
+  icon.insertAdjacentHTML("beforebegin", generateIcon(icon_data))
+  icon.parentNode.removeChild(icon);
+})
+
+function generateIcon(icon_data){
+  var icon =  "<svg " + htmlAttributes(icon_data, icon_data.options) + ">" + icon_data.path + "</svg>"
```

Comment:
You are passing icon and icon.options to htmlAttributes.
It is not consistent with htmlAttributes's [definition](https://github.com/primer/octicons/pull/242/commits/fee22773e8591520c37f46922d0f7a1a081d844a#diff-64f0a99e7e5fb7797c348ba46ce3dca8R27).

Reply:
@KartoffelCheetah Sorry my bad. I will fix it.